Frontend/Gmfctn/src/app/modules/+auth/components/sign-in/sign-in.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="wrapper">
+  <div class="content">
+    <div class="form">
+      <mat-error *ngIf="state!=''">{{state}}</mat-error>
```

Comment:
I'd suggest changing this if statement to `!!state`

Reply:
Fixed